CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# CONTRIBUTING GUIDELINES
+
+NovelRT is a community-driven project. Contribution is aslways welcome, and much appreciated.
+It is also how it'll be developed.
+
+First, please take a moment to review our [code of conduct](CODE_OF_CONDUCT.md).
+
+These guidelines are an attempt at better addressing the huge amount of pending
+issues and pull requests. Please read them closely.
+
+## Submitting Pull Requests
+
+### Getting started
```

Comment:
```suggestion
## Getting started

### Submitting Pull Requests
```

Suggestion: Reverse these? It seems to read easier this way (in the context of the flow of the document).